Model/Datasource/LdapSource.php

```diff
@@ -578,7 +590,7 @@ public function update(Model $model, $fields = null, $values = null) {
  * @param Model $model
  * @return bool Success
```

Comment:
Nice work! Some doc blocks are missing though for new attributes.

Reply:
Thanks. I forgot to run phpcs.